scripts/vm/hypervisor/xenserver/vmops

```diff
@@ -356,18 +356,25 @@ def allow_egress_traffic(session):
                 return 'false'
     return 'true'
 
+def getIpsetType():
+    try:
+        util.pread2(['/bin/bash', '-c', "ipset -v | grep 'protocol version: 6'"])
+        return 'nethash'
```

Comment:
what is the protocol version is 6, how about we test if it's less than 6 we fallback on iptreemap otherwise use the netmask type